### ISSUE TITLE
ETH missing after Withdraw attempt and ETH locked in Order

### ISSUE THREAD

**Issue by TersiaDupper** [AX-NET/axnet_issue_tracker#4]
Hi

I sent an email on the 12th but no response yet. I try to make withdraw of ETH, it say success but not show in my wallet or account anymore.
I have alsoalso a buy order of ETH locked that does not show under open orders.

Please help I'm giving back my ETH you can see my deposit, withdraw and transaction history to see how much I should have.

My username is: TersiaDupper

Thank you Sir

**Comment by superdev810** [comment]
Pls  try again for now, that issue solved

**Comment by TersiaDupper** [comment]
Nope, funds still locked in order and my previous Ethereum withdraw still missing

**Comment by superdev810** [comment]
Ok, gonna check

**Comment by realCryptoCraig** [comment]
Ok, it's been 10 days since I first reported this and my ETH from sales is still missing and I still have ETH locked in a non existent order. Did you even have a look at my account and my deposit and trade history?
user - TersiaDupper

**Comment by TersiaDupper** [comment]
Ok, did you even look at my account yet? 0.3 ETH withdraw missing and I have ETH locked in non existing order.
I have been quite patient, it has been 10 days now, please help.

user - TersiaDupper

**Comment by TersiaDupper** [comment]
Am I going to have my funds restored or should I just give up on trying to get help? My ETH I tried to withdraw is still missing and I still have ETH locked in a non existent order. Please help me

**Comment by TersiaDupper** [comment]
Ok, when are you going to resolve my problem? It's been 2 weeks and I cannot access my funds.
